### PR TITLE
update release blocking issue handling for 1.6

### PR DESCRIPTION
**Pull request by calebamiles**
notifies community that only milestone will determine whether an issue is release blocking or not

### CODE REVIEW

**Comment by calebamiles** [comment]
addresses https://github.com/kubernetes/features/pull/198#issuecomment-285186003

File: release-1.6/release-1.6.md [hunk 1]
```diff
@@ -52,6 +52,18 @@ modifications:
...
 link](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit#gid=0)
 
+## Notable operational changes
+
+1. Starting in the 1.6 release the [release team](https://github.com/kubernetes/features/blob/master/release-1.6/release_team.md)
+  will use the following procedure to identify release blocking issues
+  1. Any issues listed in the [v1.6 milestone](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6)
+     will be considered release blocking. It is everyone's responsibility to move non blocking issues out of the 1.6 milestone and into the 1.6.1 or 1.7
+     milestones **or the release will not ship**
+  1. The release team *will not* use or consider the `release-blocking` or `non-release-blocking` labels although they may have meaning
```
Comment (ethernetdan) [inline release-1.6/release-1.6.md]: Should we remove these labels then?

Comment (calebamiles) [inline release-1.6/release-1.6.md]: seems like the community still uses them

File: release-1.6/release-1.6.md [hunk 2]
```diff
@@ -52,6 +52,19 @@ modifications:
 [Feature tracking spreadsheet
 link](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit#gid=0)
 
+## Notable operational changes
+
+1. Starting in the 1.6 release the [release team](https://github.com/kubernetes/features/blob/master/release-1.6/release_team.md)
+  will use the following procedure to identify release blocking issues
+  1. Any issues listed in the [v1.6 milestone](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6)
+     will be considered release blocking. It is everyone's responsibility to move non blocking issues out of the 1.6 milestone and into the 1.6.1 or 1.7
```
Comment (luxas) [inline release-1.6/release-1.6.md]: would it make more sense to rename 1.6.1 to v1.6-nonblocking or the like?

**Pull request merged**